bootstrap.sh

```diff
@@ -332,9 +331,9 @@ case "$cmd" in
   test|test_cmds|bench|release|release_dryrun)
     $cmd "$@"
     ;;
-  "docs-release")
+  "docs-cut-version")
     build
-    docs/bootstrap.sh docs-release
+    docs/bootstrap.sh $cmd "$@"
```

Comment:
added a new script below that for now is only being called locally but can be done on a separate CI. baby steps